packages/cli/commands/project/cloneApp.js

```diff
@@ -52,8 +55,6 @@ exports.handler = async options => {
   const accountConfig = getAccountConfig(accountId);
   const accountName = uiAccountDescription(accountId);
 
-  trackCommandUsage('clone-app', {}, accountId);
```

Comment:
Should we include a tracking event early on just to mark if a customer attempts the command. Would we need to move it own after the selection of the `appId`?

Reply:
Yes, we should still call `trackCommandUsage()` here without appId. This tracking method is intended to show us stats on every command run, so we always intentionally call it at the top of our commands. The metadata tracking command lets us pass along some additional data in follow-up requests so we can dig deeper if we want to.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1127/commits/32cd50327d070cff90dc3dc3b07ec5a22bd993d8